components/add-member.tsx

```diff
@@ -60,77 +60,80 @@ export default function AddMemberForm({ user }: { user: any }) {
 
   React.useEffect(() => {
     async function fetchMemberGroups() {
-      const { data, error } = await supabase
-        .from("member_groups")
-        .select()
-        .eq("created_by", user.id);
-      if (error) {
+      try {
+        const { data, error } = await supabase
+          .from("member_groups")
+          .select()
+          .eq("created_by", user.id);
+        if (error) {
+          console.error("Error fetching member groups:", error);
+        } else {
+          
+          setSelectedGroups(null);
+          setMemberGroups(data);
+        }
+      } catch (error) {
         console.error("Error fetching member groups:", error);
-      } else {
-        console.log("member groups", data);
-        setSelectedGroups(null);
-        setMemberGroups(data);
       }
-    }
-    fetchMemberGroups();
-  }, [supabase, submitted]);
+      try {
+        fetchMemberGroups();
+      } catch (error) {
+        console.error("Error fetching member groups:", error);
+      }
```

Comment:
This code should not add new lines. It should only delete the line containing `console.log`